nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -854,17 +871,36 @@ def write_recording(
         of the format
             metadata['Ecephys'] = {}
         with keys of the forms
-            metadata['Ecephys']['Device'] = [{'name': my_name,
-                                                'description': my_description}, ...]
-            metadata['Ecephys']['ElectrodeGroup'] = [{'name': my_name,
-                                                        'description': my_description,
-                                                        'location': electrode_location,
-                                                        'device': my_device_name}, ...]
-            metadata['Ecephys']['Electrodes'] = [{'name': my_name,
-                                                    'description': my_description,
-                                                    'data': [my_electrode_data]}, ...]
-            metadata['Ecephys']['ElectricalSeries'] = {'name': my_name,
-                                                        'description': my_description}
+            metadata['Ecephys']['Device'] = [
+                {
```

Comment:
thanks, this looks better

Reply:
Thanks to Cody! I think this is all done by a bot he added. ;-)